data_structures/variables.md

```diff
@@ -64,6 +64,24 @@ The *scope* of a variable is the region of code where the variable can be access
     int variable = 2;
     int VARIABLE = 3;
     ```
+1. :star: What is the scope of variable `n` in the following code snippet?
+   ```java
+   int fac(int n) {
+       int p = 1;
+       for (int i = 1; i <= n; i++)
+           p = p * i;
```

Comment:
I prefer to ALWAYS use curly braces with loops and if statements.

Reply:
I'll fix that.

Reply:
I believe those changes are now in the main line.